store/tikv/gcworker/gc_worker.go

```diff
@@ -511,7 +526,113 @@ func resolveLocks(ctx goctx.Context, store tikv.Storage, safePoint uint64, ident
 	return nil
 }
 
-func doGC(ctx goctx.Context, store tikv.Storage, safePoint uint64, identifier string) error {
+type gcTask struct {
+	startKey  []byte
+	endKey    []byte
+	safePoint uint64
+}
+
+type gcResult struct {
+	successRegions int
+	failedRegions  int
+	lastErr        error
+}
+
+type gcTaskWorker struct {
+	store    tikv.Storage
+	taskCh   chan *gcTask
+	resultCh chan *gcResult
+}
+
+func newGCTaskWorker(store tikv.Storage, taskCh chan *gcTask, resultCh chan *gcResult) *gcTaskWorker {
+	return &gcTaskWorker{store, taskCh, resultCh}
+}
+
+func (w *gcTaskWorker) run() {
+	for {
+		task := <-w.taskCh
+		if task == nil {
+			w.resultCh <- nil
+			return
+		}
+
+		res, err := doGCForRange(w.store, task.startKey, task.endKey, task.safePoint)
+		if err != nil {
+			res = &gcResult{
+				successRegions: 0,
+				failedRegions:  0,
+				lastErr:        err,
+			}
+		}
+		if res != nil {
+			w.resultCh <- res
+		}
+	}
+}
+
+func (w *GCWorker) loadGcConcurrencyWithDefault() (int, error) {
```

Comment:
s/Gc/GC
Use capital letters for abbreviation

Reply:
ok